pyriemann/estimation.py

```diff
@@ -206,17 +206,21 @@ class XdawnCovariances(BaseEstimator, TransformerMixin):
     """
 
     def __init__(self, nfilter=4, applyfilters=True, classes=None,
-                 estimator='scm', xdawn_estimator='scm'):
+                 estimator='scm', xdawn_estimator='scm', baseline_cov=None):
         """Init."""
         self.applyfilters = applyfilters
         self.estimator = estimator
         self.xdawn_estimator = xdawn_estimator
         self.classes = classes
         self.nfilter = nfilter
+        self.baseline_cov = baseline_cov
 
     def fit(self, X, y):
+        # FIXME self.Xd should be init at XdawnCovariances.__init__ and be
+        # replaced as param not attribute
```

Comment:
I think in encountered problem when object are initialized in the init. it caused issues for `deepcopy()` of the object in scikit-learn pipeline. it's something to check because it's all over the place in the toolbox.